_posts/en/newsletters/2020-08-05-newsletter.md

```diff
@@ -0,0 +1,301 @@
+  Sapio and OP_CHECKTEMPLATEVERIFY could theoretically have been
+  utilized to prevent funds unintentionally moving to the 2-of-3
+  multisig emergency backup.  ([transcript][sapio transcript], [video][sapio video], [slides][sapio slides])
+
+- **Sydney meetup discussion:** Anonymized participants discussed
+  resolved bugs in the Bitcoin Core build system over the past months
+  and the future challenges of building and distributing Bitcoin Core
+  binaries on MacOS in light of notarization requirements and Apple
+  transitioning from Intel to ARM processors. Other topics included
+  updates to the [SIGHASH_ANYPREVOUT][topic sighash_noinput]
+  proposal, generalized Bitcoin-compatible channels, and the latest thinking
+  on Taproot activation.  ([transcript][sydney transcript])
+
+- **BIP-Taproot:** Pieter Wuille and Russell O’Connor participated in a
+  joint event organized by London BitDevs and Bitcoin Munich exploring
+  the history of how the original idea of [MAST][topic MAST] evolved
+  into the final [taproot][topic Taproot] proposal. Wuille talked about
+  how his personal motivation switched from seeking to enable cross
+  input signature aggregation to bolstering the privacy and efficiency
+  of more complex transactions. O’Connor also gave an update on
+  development of the Simplicity language (see [Newsletter #96][news96
+  simplicity]). He speculated how Simplicity could be
+  implemented as a Tapleaf version in the coming years and used for
+  delegation, [covenants][topic covenants], and other functionality not currently available using
+  Bitcoin Script. The [PR][libsecp256k1 #558] for [schnorr signatures][topic schnorr signatures] in libsecp256k1 and
+  the taproot [PR][Bitcoin Core #17977] in Bitcoin Core are seeking
+  further review and O’Connor encouraged the community to consider what
+  taproot might break in their own software well in advance of any
+  possible deployment.  ([transcript][london/munich transcript], [video][london/munich video])
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.9.0][C-Lightning 0.9.0] is the newest major version of
+  C-Lightning.  It adds support for the updated `pay` command and new
+  `keysend` RPC both described in [Newsletter #107][news107 notable].  It
+  also includes several other notable changes and multiple bug fixes.
+
+- [Bitcoin Core 0.20.1][Bitcoin Core 0.20.1] is a new maintenance
+  release.  In addition to bug fixes and some RPC behavior changes
+  resulting from those fixes, the planned release provides compatibility
+  with recent versions of [HWI][topic HWI] and its support for hardware
+  wallet firmware released for the [fee overpayment attack][].
+
+- [LND 0.11.0-beta.rc1][LND 0.11.0-beta] is the first release candidate
+  for a new major release.
```

Comment:
They never published a draft release notes and the diff was larger than I wanted to go through.  Although I'd certainly prefer to list some details, I think a short bullet for the first RC for a major release is ok.  Hopefully they have notes for next week or I'll have more time to make my own.  (Thanks for mentioning this lack of detail, though, in case it had been accidental on my part.)